DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/gzip/100b.txt

```diff
@@ -0,0 +1 @@
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
Can you move this file to a data directory and update the README simulator launch command to illustrate its use with this file?

Reply:
Ok!